Dockerfile.build

```diff
@@ -0,0 +1,3 @@
+FROM maven:3-jdk-7-onbuild
+
+CMD ["mvn test"]
```

Comment:
Why do we need this `Dockerfile.build` now that S3Proxy provides a `Dockerfile`?